### PR TITLE
:sparkles: PIC-288: Added risk register feature

### PR DESCRIPTION
**Pull request by katie-cawthorne**
- Added new endpoint to retrieve risk data from court-case-service 
- Updated risk view to display risk data
- Added mock data for registrations
- Added integration tests

### CODE REVIEW

**Review by sarahyjja — COMMENTED** [review]
👍

File: server/routes/index.js [hunk 1]
```diff
@@ -164,6 +165,16 @@ module.exports = function Index ({ authenticationMiddleware }) {
     const templateValues = await getCaseAndTemplateValues(req)
     templateValues.title = 'Risk register'
 
+    const riskResponse = await getRiskDetails(req.params.caseNo)
```
Comment (theDustRoom) [inline server/routes/index.js:168]: We need to pass the CRN to the community endpoint to retrieve risk data.

File: mappings/community/6627839278-registrations.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (theDustRoom) [inline mappings/community/6627839278-registrations.json:1]: Can we rename the mocks wit the CRN for consistency with other community mocks?

File: mappings/community/2608860141-registrations.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (theDustRoom) [inline mappings/community/2608860141-registrations.json:1]: Can we rename the mocks wit the CRN for consistency with other community mocks?

File: mappings/community/6627839278-registrations.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "id": "20e9deb2-07bb-11eb-adc1-0242ac120002",
+  "request": {
+    "method": "GET",
+    "urlPathPattern": "/offender/6627839278/registrations"
```
Comment (theDustRoom) [inline mappings/community/6627839278-registrations.json:5]: Change case number to the CRN.

File: mappings/community/2608860141-registrations.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  "id": "20e9deb2-07bb-11eb-adc1-0242ac120002",
+  "request": {
+    "method": "GET",
+    "urlPathPattern": "/offender/2608860141/registrations"
```
Comment (theDustRoom) [inline mappings/community/2608860141-registrations.json:5]: Change case number to the CRN.

**Review by theDustRoom — CHANGES_REQUESTED** [review]

**Comment by theDustRoom** [comment]
Also noted that there are no Swagger validation tests for the new risk endpoint mocks.

File: server/routes/index.js (hunk 1 above)
Comment (katie-cawthorne) [reply server/routes/index.js:168]: Yep, I'll sort

File: mappings/community/6627839278-registrations.json (hunk 2 above)
Comment (katie-cawthorne) [reply mappings/community/6627839278-registrations.json:1]: Yes sure, makes sense

**Review by theDustRoom — APPROVED** [review]
Nice job 👍

**Pull request merged**